<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_Equipo_1</class>
 <widget class="QMainWindow" name="Login_Equipo_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login_Equipo_1</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(5, 13, 33);
font: 10pt &quot;Arial&quot;;</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="layout2" stretch="0,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="info">
        <property name="minimumSize">
         <size>
          <width>158</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>158</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(201, 213, 238);</string>
        </property>
        <property name="text">
         <string>¿No tiene una cuenta?</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::Qt::AlignmentFlag::AlignVCenter">
       <widget class="QPushButton" name="button_login_1">
        <property name="minimumSize">
         <size>
          <width>157</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>157</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(26, 103, 248);</string>
        </property>
        <property name="text">
         <string>Crear Cuenta</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="button_login_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>460</width>
        <height>51</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(26, 103, 248);
</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="titulo_main">
      <property name="minimumSize">
       <size>
        <width>67</width>
        <height>160</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(201, 213, 238);
 font-size:20pt;
</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QFormLayout" name="layout1">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="titulo_usuario">
        <property name="minimumSize">
         <size>
          <width>121</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>121</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(201, 213, 238);</string>
        </property>
        <property name="text">
         <string>Usuario</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="campo_usuario">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>161</width>
          <height>51</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>lorenzo29</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="titulo_password">
        <property name="minimumSize">
         <size>
          <width>121</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>121</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(201, 213, 238);</string>
        </property>
        <property name="text">
         <string>Contraseña</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="campo_password">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>161</width>
          <height>51</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string>duhfhuhdufh</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>titulo_main</zorder>
   <zorder>button_login_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(201, 213, 238);</string>
   </property>
   <widget class="QMenu" name="menuEditar">
    <property name="styleSheet">
     <string notr="true">color: rgb(201, 213, 238);</string>
    </property>
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionDeshacer"/>
    <addaction name="separator"/>
    <addaction name="actionReahacer"/>
    <addaction name="separator"/>
    <addaction name="actionCortar"/>
    <addaction name="separator"/>
    <addaction name="actionCopiar"/>
    <addaction name="separator"/>
    <addaction name="actionPegar"/>
   </widget>
   <widget class="QMenu" name="menuVista">
    <property name="styleSheet">
     <string notr="true">color: rgb(201, 213, 238);</string>
    </property>
    <property name="title">
     <string>Vista</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_out"/>
   </widget>
   <widget class="QMenu" name="menuArchivo">
    <property name="styleSheet">
     <string notr="true">color: rgb(201, 213, 238);</string>
    </property>
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
    <addaction name="separator"/>
    <addaction name="actionGuardar"/>
    <addaction name="separator"/>
    <addaction name="actionGuardar_Como"/>
    <addaction name="separator"/>
    <addaction name="actionEditar"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="styleSheet">
     <string notr="true">color: rgb(201, 213, 238);</string>
    </property>
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionDocumentacion_2"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_2"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuVista"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionpepe">
   <property name="text">
    <string>pepe</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionGuardar_Como">
   <property name="text">
    <string>Guardar Como</string>
   </property>
  </action>
  <action name="actionEditar">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionDeshacer">
   <property name="text">
    <string>Deshacer</string>
   </property>
  </action>
  <action name="actionReahacer">
   <property name="text">
    <string>Reahacer</string>
   </property>
  </action>
  <action name="actionCortar">
   <property name="text">
    <string>Cortar</string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="text">
    <string>Copiar</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionDocumentacion">
   <property name="text">
    <string>Documentacion</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDocumentacion_2">
   <property name="text">
    <string>Documentacion</string>
   </property>
  </action>
  <action name="actionAbout_2">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPegar">
   <property name="text">
    <string>Pegar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
